front/public/locales/fr/stdcm.json

```
@@ -49,6 +50,14 @@
     "anterior": "Indiquer le sillon antérieur",
     "posterior": "Indiquer le sillon postérieur"
   },
+  "mailFeedback": {
+    "body": "%0A********%0A%0ANous%20vous%20invitons%20à%20nous%20envoyer%20un%20message.%0ADès%20que%20vous%20avez%20un%20gêne%20ou%20une%20remarque,%0Amême%20si%20vous%20n’y%20mettez%20pas%20la%20forme.%0A%0AMerci%20à%20vous%20par%20avance,%0AL’équipe%20du%20projet%20LMR%0A%0A(vous%20pouvez%20effacer%20ce%20message)%0A%0A********",
```

The translation key contains formatting that should be handled in the component:

This way:
- Translation file only contains content
- Layout/formatting is managed in the component
- Easier to maintain and update translations

```suggestion
    "body": "Nous vous invitons à nous envoyer un message.\nDès que vous avez une gêne ou une remarque,\nmême si vous n'y mettez pas la forme.\n\nMerci à vous par avance,\nL'équipe du projet LMR\n\n(vous pouvez effacer ce message)"
```

---

done

---

you forgot to change the english version too ;)

---

Ooooh my bad 🤦‍♂️

---

done